spec.html

```diff
@@ -39304,14 +39321,12 @@ <h1>
           <emu-alg>
             1. Let _srcData_ be _srcArray_.[[ViewedArrayBuffer]].
             1. If IsDetachedBuffer(_srcData_) is *true*, throw a *TypeError* exception.
-            1. Let _constructorName_ be the String value of _O_.[[TypedArrayName]].
-            1. Let _elementType_ be the Element Type value in <emu-xref href="#table-the-typedarray-constructors"></emu-xref> for _constructorName_.
+            1. Let _elementType_ be TypedArrayElementType(_O_).
             1. Let _elementLength_ be _srcArray_.[[ArrayLength]].
-            1. Let _srcName_ be the String value of _srcArray_.[[TypedArrayName]].
-            1. Let _srcType_ be the Element Type value in <emu-xref href="#table-the-typedarray-constructors"></emu-xref> for _srcName_.
-            1. Let _srcElementSize_ be the Element Size value specified in <emu-xref href="#table-the-typedarray-constructors"></emu-xref> for _srcName_.
+            1. Let _srcType_ be TypedArrayElementType(_srcArray_).
+            1. Let _srcElementSize_ be TypedArrayElementSize(_srcArray_).
             1. Let _srcByteOffset_ be _srcArray_.[[ByteOffset]].
-            1. Let _elementSize_ be the Element Size value specified in <emu-xref href="#table-the-typedarray-constructors"></emu-xref> for _constructorName_.
+            1. Let _elementSize_ be TypedArrayElementSize(_O_).
```

Comment:
We could group these bindings a little more logically if you like.

Reply:
Done.